api/v1beta1/authorino_types.go

```diff
@@ -0,0 +1,139 @@
+/*
+Copyright 2021.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import (
+	apiv1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+type ConditionType string
+
+const (
+	// ConditionReady specifies that the resource is ready
+	ConditionReady             ConditionType = "Ready"
+	AuthorinoContainerName     string        = "authorino"
+	AuthorinoOperatorNamespace string        = "authorino-operator"
+
+	// Authorino EnvVars
+	ExtAuthGRPCPort         string = "EXT_AUTH_GRPC_PORT"
+	TSLCertPath             string = "TLS_CERT"
+	TSLCertKeyPath          string = "TLS_CERT_KEY"
+	OIDCHTTPPort            string = "OIDC_HTTP_PORT"
+	OIDCTSLCertPath         string = "OIDC_TLS_CERT"
+	OIDCTLSCertKeyPath      string = "OIDC_TLS_CERT_KEY"
+	AuthConfigLabelSelector string = "AUTH_CONFIG_LABEL_SELECTOR"
```

Comment:
Attention to https://github.com/Kuadrant/authorino/pull/179, changing the name of this env var to `SECRET_LABEL_SELECTOR`, and introducing `AUTH_CONFIG_LABEL_SELECTOR`.

Reply:
I left they both there to make it backwards compatible with previous version authorino, do you reckon we should remove this?

Reply:
I think it depends on how we're going to tie versions of the operator to versions of the service, but I guess leaving both env vars is good enough for now.

Let's add a comment
```go
// TODO: remove after Authorino v0.5.0
```